src/pkg/utils/io.go

```diff
@@ -62,6 +62,13 @@ func CreateDirectory(path string, mode os.FileMode) error {
 	return nil
 }
 
+// CreateFile creates an empty file at the given path.
+func CreateFile(filepath string) error {
+	f, err := os.Create(filepath)
+	defer f.Close()
+	return err
+}
```

Comment:
nit: call this `Touch` instead as it closely mirrors the `touch` CLI

Reply:
I disagree - `CreateFile` is likely a better name given we already have `CreateDirectory` and `CreatePathAndCopy` - I would say that we should probably match the functionality in `CreateDirectory` of checking `InvalidPath(path)`in the function.

Reply:
If we do go with `Touch` we should update everything to the POSIX equivalent

Reply:
tbh I am not opposed to that. I do agree it would break consistency with what is already there, so I will rescind my claim

Reply:
After thinking about it more, changing it to a request changes (for the invalid path stuff InvalidPath(path))